server/store/storemodels/storemodels.go

```diff
@@ -2,6 +2,12 @@ package storemodels
 
 import "time"
 
+type Stats struct {
+	ConnectedUsers int64
+	SyntheticUsers int64
+	LinkedChannels int64
+}
+
```

Comment:
Moved this to the `storemodels` even though it's not persisted to simplify the import story.